src/javascript/app/pages/trade/duration.js

```diff
@@ -184,7 +184,7 @@ const Durations = (() => {
         const date_start     = CommonFunctions.getElementById('date_start').value;
         const now            = !date_start || date_start === 'now';
         const current_moment = moment((now ? window.time : parseInt(date_start) * 1000)).add(5, 'minutes').utc();
-        let expiry_date      = Defaults.get('expiry_date') ? moment(Defaults.get('expiry_date')) : current_moment;
+        let expiry_date      = Defaults.get('expiry_date') ? moment(Defaults.get('expiry_date')) : current_moment.add(1, 'days').utc();
```

Comment:
Adjust end date so that user can see the valid end-time

Reply:
this will add one day for all trade types, but some trades are fine to be on the same day. we need a better solution to only add one day for contracts that only allow daily duration.